src/ol_infrastructure/applications/xpro/__main__.py

```diff
@@ -224,11 +224,15 @@
     )
 
 # Heroku App configuration
+# env_name is 'ci' 'rc' or 'production'
+env_name = stack_info.name.lower() if stack_info.name != "QA" else "rc"
 
 # Values that are generally unchanging across environments
 heroku_vars = {
+    "AWS_STORAGE_BUCKET_NAME": f"ol-xpro-app-{env_name}",
```

Comment:
You should just be able to use the existing bucket name var from further up in the file where the bucket gets created.